backends/rapidpro/backend.go

```diff
@@ -47,11 +47,11 @@ func init() {
 }
 
 // GetChannel returns the channel for the passed in type and UUID
-func (b *backend) GetChannel(ctx context.Context, ct courier.ChannelType, uuid courier.ChannelUUID) (courier.Channel, error) {
+func (b *backend) GetChannel(ctx context.Context, ct courier.ChannelType, uuid courier.ChannelUUID, isActive bool) (courier.Channel, error) {
 	timeout, cancel := context.WithTimeout(ctx, backendTimeout)
 	defer cancel()
 
-	return getChannel(timeout, b.db, ct, uuid)
+	return getChannel(timeout, b.db, ct, uuid, isActive)
```

Comment:
do we need to get deleted channels? Can't we just treat this as "courier can't find the channel, courier fails the message"

Reply:
I wanted to be sure that is for deleted channels only

Reply:
And we need a channel set to be able to write the status too